# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sysvinit/distro.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- sysvinit-2.86/src/Makefile	2006-11-13 18:23:34.000000000 -0800
+++ sysvinit-2.86/src/Makefile	2006-11-13 19:36:36.000000000 -0800
@@ -23,6 +23,7 @@
 MAN8	= halt.8 init.8 killall5.8 pidof.8 poweroff.8 reboot.8 runlevel.8
 MAN8	+= shutdown.8 telinit.8
 
+# Unused in T2 but defined by default
 ifeq ($(DISTRO),)
 BIN	+= mountpoint
 SBIN	+= sulogin bootlogd
@@ -31,6 +32,35 @@
 MAN8	+= sulogin.8 bootlogd.8
 endif
 
+# Clean Setup for default T2 *These override*
+ifeq ($(DISTRO),T2)
+BIN	+= 
+SBIN	= sulogin init halt shutdown runlevel killall5
+USRBIN	= wall last mesg
+MAN1	= last.1 lastb.1 mesg.1 wall.1
+MAN5	= initscript.5 inittab.5
+MAN8	= halt.8 init.8 killall5.8 pidof.8 poweroff.8 reboot.8 runlevel.8 shutdown.8 telinit.8 sulogin.8
+endif
+
+# Clean Setup for default T2 without sysV init *These override*
+ifeq ($(DISTRO),T2-no-init)
+BIN	+= 
+SBIN	= sulogin halt shutdown killall5
+USRBIN	= wall last mesg
+MAN1	= last.1 lastb.1 mesg.1 wall.1
+MAN5	= initscript.5 inittab.5
+MAN8	= halt.8 killall5.8 pidof.8 poweroff.8 reboot.8 shutdown.8 telinit.8 sulogin.8
+endif
+
+# Clean Setup for Familiar 
+ifeq ($(DISTRO),familiar)
+BIN	+= mountpoint
+SBIN	= bootlogd init runlevel
+USRBIN += wall
+MAN1	+= mountpoint.1 wall.1
+MAN8	= bootlogd.8
+endif
+
 ifeq ($(DISTRO),Debian)
 BIN	+= mountpoint
 SBIN	+= sulogin bootlogd
